Flipper/Packages/UI/Sources/Device/UpdateCard/Components/UpdateWhatsNewButton.swift

```diff
@@ -34,17 +25,21 @@ struct UpdateWhatsNewButton: View {
     }
 
     var body: some View {
-        if let state = readyState, let firmware = updateModel.firmware {
+        if
```

Comment:
I would move this condition out of the button to DeviceUpdateCard `isWhatsNewVisible: Bool { ... }` and use opacity modifier on UpdateWhatsNewButton()